packages/core/src/messages/responses/response-base.ts

```diff
@@ -36,7 +36,7 @@ export interface IResponseSubscription {
 }
 
 export abstract class BaseResponseError extends ResponseBase {
-  public _innerException: SdkError;
+  private readonly _innerException: SdkError;
```

Comment:
The CI fails on making `inner_exception` field private. It might be getting accessed somewhere in the sdk/examples?